server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/CreateAction.java

```diff
@@ -82,6 +88,10 @@ public void define(WebService.NewController controller) {
       .setResponseExample(getClass().getResource("example-create.json"))
       .setHandler(this);
 
+    qProfileWsSupport
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2209)